.github/ci/docs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
These scripts might be located as functions inside `common.sh`.

Reply:
As an initial support for running CI on GH actions I think we may keep them separated. Mainly because there are some of them which require running different targets.

I think having the steps defined in each of the CI scripts makes it easier to track which CI is executing which target